examples/gno.land/r/demo/foo20-airdrop/airdrop_test.gno

```diff
@@ -0,0 +1,65 @@
+package foo20airdrop
+
+import (
+	"std"
+	"testing"
+
+	"gno.land/p/demo/airdrop"
+	"gno.land/p/demo/merkle"
+	"gno.land/r/demo/foo20"
+	"gno.land/r/demo/users"
+)
+
+var leaves []merkle.Hashable = []airdrop.AirdropData{
+	{
+		Address: "g1auhc2cymv7gn9qmls0ttdr3wqrljgz0dhq90e",
+		Amount:  1_000_000,
+	},
+	{
+		Address: "g1zyvskpxg5lv4qpygtuvp93zprrrjpk2exa9rfx",
+		Amount:  1_000_000,
+	},
+	{
+		Address: "g14szvkruznx49sxe4m9dmg3m8606sm6yp4a0wv8",
+		Amount:  1_000_000,
+	},
+}
+
+func TestRegisterMerkle(t *testing.T) {
```

Comment:
What do you think about completely removing this test?
You verify it works in `TestClaimAirdrop` regardless (happy path)